pkg/rtc/room.go

```diff
@@ -1741,12 +1747,37 @@ func (r *Room) createAgentDispatchesFromRoomAgent() {
 	}
 }
 
+func (r *Room) IsDataMessageUserPacketDuplicate(up *livekit.UserPacket) bool {
+	r.lock.Lock()
```

Comment:
defer unlock?

Reply:
Thank you. Done in https://github.com/livekit/livekit/pull/3366/commits/44b439476c5a44f55471550bffe4432360792612